saos-webapp/src/main/resources/message/common.properties

```diff
@@ -83,6 +83,13 @@ month.11 = november
 month.12 = december
 
 
+/* Cookie policy */
+cookie.policy.previewMessage = Polityka cookies
+cookie.policy.show = Zobacz
+cookie.policy.message = Ta strona wykorzystuje pliki cookies (niewielkie pliki tekstowe przechowywane przez przeglądarkę internetową na urządzeniu użytkownika) m.in. do analizy statystycznej ruchu, dopasowania wyglądu i treści strony do indywidualnych potrzeb użytkownika. Pozostawiając w ustawieniach przeglądarki włączoną obsługę plików cookies wyrażasz zgodę na ich użycie. Jeśli nie zgadzasz się na wykorzystanie plików cookies zmień ustawienia swojej przeglądarki.
```

Comment:
What do you mean by "statystyczna analiza ruchu"? Future analytics tools like google analythics for example?